foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java

```diff
@@ -161,39 +161,39 @@ public static AbstractConfiguration convertEnvVariable(AbstractConfiguration sou
     return source;
   }
 
-  //inject a copy of cse.xxx for servicecomb.xxx
-  private static void duplicateServiceCombConfigToCse(AbstractConfiguration source) {
+  //inject a copy of servicecomb.xxx for cse.xxx
+  private static void duplicateCseConfigToServicecomb(AbstractConfiguration source) {
```

Comment:
Just a quick question，if the user has two value setting for servicecomb.xxx and cse.xxx, which one will take effect？
We may need add some document for it.

Reply:
Depends on servicecomb-config-order, and I add new test cased for this in new commit. And adding warning message if two keys are in different files.